README.md

```diff
@@ -42,6 +42,11 @@ By default, the websocket port 4233 is used, you can change it via:
 export SIGNALTOWER_PORT=1234
 ```
 
+By default, the websocket is bound to all interfaces (0.0.0.0), you can also bind it just localhost (127.0.0.1) via:
+```
+export SIGNALTOWER_LOCALHOST
```

Comment:
Does just exporting an ENV variable set it to true, or is the documentation line incomplete?

Reply:
It does not set a value to the ENV variable but it does set it. Document it as
`export SIGNALTOWER_LOCALHOST True` could suggest that you can also set it `False` which is not the case.